lib/simp/cli/puppetfile/skeleton.rb

```diff
@@ -0,0 +1,58 @@
+require 'simp/cli'
+module Simp::Cli::Puppetfile
+  # Provides a skeleton Puppetfile that can include a local Puppetfile.simp
+  class Skeleton
+    # @return [String] all modules' data as a Puppetfile
+    def self.to_puppetfile
+      <<-PUPPETFILE.gsub(/ {8}/,'')
+        # ==============================================================================
+        # SIMP Puppet modules
+        # ------------------------------------------------------------------------------
+        # The line below enables this Puppetfile to deploy all of SIMP's modules from a
+        # neighboring `Puppetfile.simp` file.
+        #
+        # If you install SIMP modules locally from RPMs, you can generate a current
+        # `Puppetfile.simp` at any time by running the command:
+        #
+        #     simp puppetfile generate > Puppetfile.simp
+        #
+        # You can regenerate a clean copy of this Puppetfile at any time by running:
+        #
+        #     simp puppetfile generate --skeleton > Puppetfile
+        #
+        # ------------------------------------------------------------------------------
+        instance_eval(File.read("Puppetfile.simp"))
```

Comment:
Probably should explain this

Reply:
Also, do we need to check and see if that file exists before loading it? Not sure how much Ruby we want to stuff in this thing.

Reply:
> Probably should explain this

I couldn't think of a concise way to explain the technical implementation of this line for Ruby nerds<sup>*</sup>) in a way that wouldn't also obfuscate the basic purpose of and usage of the file for general users.  This is something we can tackle in a follow-on ticket if we think of a better way.

(* = basically, `require` and `load` won't work while r10k is processing the file, so you need to do this)



> Also, do we need to check and see if that file exists before loading it? Not sure how much Ruby we want to stuff in this thing.

This `Puppetfile` is only generated by `generate --skeleton` in order to use `Puppetfile.simp`—I considered wrapping it in additional logic to spit out a prettier error message, but seemed to add more clutter than it was worth to an already-busy Puppetfile.